lgc/test/BuiltIns/cs-globalinvocationid.lgc

```diff
@@ -1,7 +1,6 @@
 ; NOTE: Assertions have been autogenerated by tool/update_llpc_test_checks.py UTC_ARGS: --tool lgc
 ; RUN: lgc -mcpu=gfx1010 - < %s | FileCheck --check-prefixes=GFX8 %s
```

Comment:
This fooled me by checking for GFX8 while it actually compiled with gfx10.1

Reply:
Apparently this was not correctly updated in aff7949fdd.
There seem to be more affected tests, I guess we should also fix them if we fix this one.

Reply:
https://github.com/GPUOpen-Drivers/llpc/pull/2848